app/gui2/shared/binaryProtocol.ts

```diff
@@ -0,0 +1,2217 @@
+// automatically generated by the FlatBuffers compiler, and modified
+// Helper classes copied from https://github.com/google/flatbuffers,
+// which is licensed under the Apache License, Version 2.0:
```

Comment:
Could you describe a bit more how it was generated?
- the tool and version used (i assume flatc, but not sure which one)
- what kind modifications were made